Library/Homebrew/sandbox.rb

```diff
@@ -91,6 +91,32 @@ def deny_write_homebrew_repository
     end
   end
 
+  sig { params(path: T.any(String, Pathname), type: T.nilable(Symbol)).void }
```

Comment:
Why do these need to be `nilable`?

Reply:
You're right, these don't need to be `nilable` (copied/adapted from some temporary sig I had for `allow_write`, which _does_ need `nilable` in its current state).